components/NavigationDrawer/tests/snapshot/MDCBottomDrawerControllerSnapshotTests.m

```diff
@@ -152,4 +152,30 @@ - (void)testPresentedDrawerWithColoredViewsWithVerticalSizeClassCompact {
   [self generateSnapshotAndVerifyForView:self.bottomDrawerViewController.view];
 }
 
+- (void)testPresentedDrawerWithShadowElevation {
```

Comment:
Why did only `testPresentedDrawerWithColoredViews` receive a shadow/elevation? I would have expected the default behavior to include elevation after this change.

Reply:
The snapshots did get an update due to the new default elevation